lib/workers/repository/update/branch/execute-post-upgrade-commands.ts

```diff
@@ -98,6 +98,19 @@ export async function postUpgradeCommandsExecutor(
         }
       }
 
+      const workingDirTemplate = upgrade.postUpgradeTasks?.workingDirTemplate;
+      let workingDir: string | null = null;
+
+      if (workingDirTemplate) {
+        workingDir = sanitize(
```

Comment:
ensure it's inside local dir, use this is util 

https://github.com/renovatebot/renovate/blob/5ec2a138261857776a8c9ee2b204b2d4c2b213f8/lib/util/fs/index.ts#L116